### PR TITLE
Drop schema version history tables

### PR DESCRIPTION
**Pull request by ryanrath**
## Description
The `schema_version_history` table is not being used currently in an effort to clean up the code base it is being removed here. 

## Motivation and Context
These changes originally arose from work I was doing to make sure that XDMoD can run on CentOS8 w/ newer versions of PHP & MariaDB. Specifically, I ran into problems when the ETL process would load the default data into these tables due to the data having a value of `NOW()` for the `action_datetime` column. This was caused by a change MariaDB made to be more strict in how it deals with incoming data so whereas in the past it would coerce `'NOW()'` ( notice the single quotes ) to `NOW()`, in more recent versions it does not coerce the string to a function call and instead treats it as a string. 

At first I attempted to update `action_datetime` to have a default value of `CURRENT_TIMESTAMP` ( and removing the `action_datetime` column from the data files. ) and while this worked well for the version of MariaDB available on CentOS8, the version on CentOS7 does not support `datetime` columns having a default value of `CURRENT_TIMESTAMP` ( this wouldn't be added until MySQL 5.6 or MariaDB 10.0 ). Upon discussion w/ @jtpalmer and @jpwhite4 it was decided that since this table wasn't currently being used it might as well be removed. 

## Tests performed
Automated tests were run locally on CentOS7 / 8. 

## Checklist:

- [X] The pull request description is suitable for a Changelog entry
- [X] The milestone is set correctly on the pull request
- [X] The appropriate labels have been added to the pull request

### CODE REVIEW

File: db/data/mod_logger.sql [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;
-
-/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
-/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
-/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
-/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
-/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
-/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
-/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;
-
```
Comment (jtpalmer) [inline db/data/mod_logger.sql:33]: I think this is still needed.

**Review by jtpalmer — CHANGES_REQUESTED** [review]
Please remove the `getDatabaseVersion` function from `classes/Xdmod/Version.php`.  It isn't used anywhere that I can find.  Also please update the description of the pull request.

File: db/data/mod_logger.sql (hunk 1 above)
Comment (ryanrath) [reply db/data/mod_logger.sql]: Yup, just replaced it, minus the schema_version_history insert at the end.

File: configuration/etl/etl_sql.d/migrations/9.5.0-10.0.0/drop_schema_version_history.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-- DROP the schema_version_history table in mod_hpcdb, mod_logger, moddb, modw
```
Comment (jtpalmer) [inline configuration/etl/etl_sql.d/migrations/9.5.0-10.0.0/drop_schema_version_history.sql:1]:
```suggestion
-- DROP the schema_version_history table in mod_hpcdb, mod_logger, mod_shredder, moddb, modw
```

I'm against comments like this, but if you're going to include it then might as well include all the schemas.

File: configuration/etl/etl.d/xdmod-migration-9_5_0-10_0_0.json [hunk 3]
```diff
@@ -203,39 +220,7 @@
...
-            "endpoints": {
-                "destination": {
-                    "type": "mysql",
-                    "name": "XDMoD Data Warehouse",
-                    "config": "datawarehouse",
-                    "schema": "mod_shredder"
-                }
-            }
-        },
+         },
```
Comment (jtpalmer) [inline configuration/etl/etl.d/xdmod-migration-9_5_0-10_0_0.json:223]:
```suggestion
        },
```

More indentation that shouldn't change.

File: classes/Xdmod/Version.php [hunk 4]
```diff
@@ -35,14 +35,7 @@ class Version
...
 
-    /**
-     * Name of the schema version history table.
-     *
-     * @var string
-     */
-    protected static $versionTableName = 'schema_version_history';
-
-    /**
+      /**
```
Comment (jtpalmer) [inline classes/Xdmod/Version.php:38]:
```suggestion
    /**
```
Indentation shouldn't change here.

File: configuration/etl/etl_sql.d/migrations/9.5.0-10.0.0/drop_schema_version_history.sql (hunk 2 above)
Comment (ryanrath) [reply configuration/etl/etl_sql.d/migrations/9.5.0-10.0.0/drop_schema_version_history.sql:1]: agreed about including all of the tables in the comment. What kind of comment would you prefer to see here?

Comment (jtpalmer) [reply configuration/etl/etl_sql.d/migrations/9.5.0-10.0.0/drop_schema_version_history.sql:1]: I don't think any comment is necessary.  If you really need one you could explain why the tables are being dropped.

**Review by jtpalmer — APPROVED** [review]
Please update the description of the pull request.

**Pull request merged**